sympy/stats/frv_types.py

```diff
@@ -33,13 +34,16 @@
 'Binomial',
 'BetaBinomial',
 'Hypergeometric',
-'Rademacher'
+'Rademacher',
+'IdealSoliton',
+'RobustSoliton',
 ]
 
-def rv(name, cls, *args):
+def rv(name, cls, *args, **kwargs):
     args = list(map(sympify, args))
     dist = cls(*args)
-    dist.check(*args)
+    if kwargs.pop('check', True):
+        dist.check(*args)
```

Comment:
Why this change?

Reply:
This change is already in master. I think its showing up because it was not present when I opened the PR initially and now I synchronized with master